src/custom/components/CowBalance/index.tsx

```diff
@@ -15,6 +15,7 @@ export interface CowBalanceProps {
   account?: string
   balance?: CurrencyAmount<Currency>
   title?: string
+  subsidy: { tier: number; discount: number }
```

Comment:
If you give a name to this type, you then can make it more readable below and not use `Pick<CowBalanceProps, 'subsidy'>`